fastify.js

```diff
@@ -266,7 +267,7 @@ function fastify (options) {
     },
     version: {
       get () {
```

Comment:
Because none of those are instance specific but they are module specific. The right scope is the module.
(Having them for each instance will have the load happen every test for example, and it counts a few CPU cycle).

Reply:
this has been reviewed like so: https://github.com/fastify/fastify/pull/2549#discussion_r486364101

Reply:
Okay.